File: function/notes.py
# Ryan Crop, Function Notes in python

#Function is an action stored in a keyword

#snake_Case = no capitals, underscores instead of spaces
#camalCase = removes spaces, any word AFTER the 1st starts with a capital 
#number = int(input("Tell me a number:\n"))
#def add(numOne, numTwo): #Parameters go in the parenthesis seperated by commas
    #print(numOne+numTwo)

#add(number,20)#Arguments are given when the function is called and must match the number of parameters

def user(word):
   return input(f"Tell me a {word}:\n")

name = user("name")
verb = user("verb")
place = user("place")
print(f"{name} was {verb} and somehow got to {place}")

File: all_of_python_and_c/function/notes.c
//Ryan Crop, Function Notes in c
#include <stdio.h>

//void add(int numOne,int numTwo){
    //printf("%d\n", numOne+numTwo);
//}
void input(char type[50], int length){
    char answer[50];
    printf("Please give me a %s\n", type);
    getStr(answer, sizeof(answer)-1);
    return answer;
}



int main(void){
//printf("Hello World\n");
//add(4,9);
//add(1000,2);
//add(47,9);
//add(82,-9);
char name = word("name");
printf("%s was %s until they somehow reached %s", word("name", 50), word("verb", 50), word("place", 50));
    return 0;
}